cli/src/common/utils.py

```diff
@@ -43,3 +46,32 @@ def validate_json_format(data):
     except ValueError:
         return False
     return True
+
+
+def get_bucket_object_name(url):
```

Comment:
let's split this in two method, one to get the scheme and one to parse a S3 Url to get the bucket name and the key name

Reply:
Done, remove the https case, it's useless in current cases, add full object name as return value